tests/stockfish/test_models.py

```diff
@@ -408,3 +408,26 @@ def test_make_moves_transposition_table_speed(self):
             total_time_calculating_second += default_timer() - start
 
         assert total_time_calculating_first < total_time_calculating_second
+
+    def test_benchmark(self, stockfish):
+        defaults = stockfish.benchmark()
```

Comment:
Let's rename `default` to `benchmark_result_with_defaults` for example

Also two other result below

Reply:
Fixed

Reply:
Maybe it would be better to call it `result` in every test case?

```python
results = stockfish.benchmark()
```